cli/Fixtures/app_with_spm_dependencies/Package.swift

```diff
@@ -8,5 +8,6 @@ let package = Package(
         .package(url: "https://github.com/Quick/Nimble", exact: "13.2.0"),
         .package(url: "https://github.com/firebase/firebase-ios-sdk.git", exact: "10.21.0"),
         .package(url: "https://github.com/getsentry/sentry-cocoa.git", exact: "8.26.0"),
+        .package(url: "https://github.com/lzell/AIProxySwift", exact: "0.130.0"),
```

Comment:
we can go ahead with this, but the dependencies acceptance tests are already slow as-is. When we want to test a very specific feature of SwiftPM like in this case, I'd consider creating a local package instead of including an actual package.

Reply:
I reverted this to use a local package instead.